publish.sh

```diff
@@ -31,13 +23,13 @@ else
     export PYPI_PACKAGE_REPOSITORY="test"
 fi
 
-# export version
-VERSION_LAST_TAG=$(git describe --abbrev=0 --tags 2>/dev/null)
 
-if [ -z "$TRAVIS_TAG" ]; then
-    export PYPI_PACKAGE_VERSION=${VERSION_LAST_TAG}-rc.${TRAVIS_BUILD_NUMBER}
-else 
-    export PYPI_PACKAGE_VERSION=${TRAVIS_TAG}
+if [[ "${EFFECTIVE_BRANCH}" =~ ^(v[0-9]+(\.[0-9]+){0,4})$ ]]; then
```

Comment:
Given the 3-decimal versioning used for this repo, instead of `(\.[0-9]+){0,4}` shouldn't it be `(\.[0-9]+){2}`